### ISSUE TITLE
No kernel name found

### ISSUE THREAD

**Issue by ijakas-syn** [google-github-actions/run-vertexai-notebook#22]
### TL;DR

Vertex AI training job fails because there is on kernel name. The kernel name is supposed to be set trough the kernel input, but is not forwarded to the step because of different variable names.

The input is:
  kernel_name:
    description: 'Notebook kernel to use for the execution environment - defaults to python3'
    default: 'python3'
    required: false

and the job references:
kernel: '${{ inputs.kernel }}'

### Expected behavior

The input is:
  kernel:
    description: 'Notebook kernel to use for the execution environment - defaults to python3'
    default: 'python3'
    required: false

and the job references:
kernel: '${{ inputs.kernel }}'

### Observed behavior

Referencing a `kernel` variable that isn't defined.

### Action YAML

```yaml
name: Execute notebooks found in a pull request and create links to their output files

on:
  pull_request:
    types: [ labeled ]
    branches:
    - main

permissions:
  pull-requests: 'write'
  issues: 'write'
  contents: write
  id-token: 'write'

env:
  GCS_SOURCE: bucket/source        # TODO: Update to a bucket with proper ACLs
  GCS_OUTPUT: bucket/end       # TODO: Update to a bucket with proper ACLs

jobs:

  # JOB to run change detection and build the allowlist
  changes:
    if: ${{ github.event.label.name == 'notebook-review' }}
    runs-on: ubuntu-latest
    # Set job outputs to values from filter step
    outputs:
      notebooks: ${{ steps.filter.outputs.notebooks }}
      notebooks_files: ${{ steps.filter.outputs.notebooks_files }}
    steps:
    # For pull requests it's not necessary to checkout the code
    - uses: dorny/paths-filter@b2feaf19c27470162a626bd6fa8438ae5b263721
      id: filter
      with:
        list-files: shell
        filters: |
          notebooks:
            - '**.ipynb'
  # JOB to initiate the notebook review process
  notebook-review:
    name: 'Notebook Review'
    needs: changes
    if: ${{ needs.changes.outputs.notebooks == 'true' }}
    runs-on: ubuntu-latest

    steps:
    - id: 'checkout'
      uses: 'actions/checkout@v3'
    # Configure Workload Identity Federation and generate an access token.
    # - id: 'auth'
    #   name: 'Authenticate to Google Cloud'
    #   uses: 'google-github-actions/auth@b258a9f230b36c9fa86dfaa43d1906bd76399edb'
    #   with:
    #     workload_identity_provider: 'projects/123456789/locations/global/workloadIdentityPools/my-pool/providers/my-provider'
    #     service_account: 'my-service-account@my-project.iam.gserviceaccount.com'

    # Alternative option - authentication via credentials json
    - id: 'auth'
      name: 'Authenticate to Google Cloud'
      uses: 'google-github-actions/auth@b258a9f230b36c9fa86dfaa43d1906bd76399edb'
      with:
        credentials_json: '${{ secrets.GCP_CREDENTIALS }}'

    - id: notebook-review
      uses: 'google-github-actions/run-vertexai-notebook@v0'
      with:
        gcs_source_bucket: '${{ env.GCS_SOURCE }}'
        gcs_output_bucket: '${{ env.GCS_OUTPUT }}'
        allowlist: '${{ needs.changes.outputs.notebooks_files }}'
```

### Log output

```text
raise ValueError("No kernel name found in notebook and no override provided.")
```

### Additional information

Thank you for developing this action!

**Comment by bradegler** [comment]
I've updated the input value to be the correct name in #23 which should resolve this issue.

**Comment by ijakas-syn** [comment]
Thank you for the update, I will try it out when you merge it.

**Comment by ijakas-syn** [comment]
Hi, I tried to run it several times after the update but there is still no kernel under the `--kernel-name` argument in Vertex.

This are the arguments under Arguments (Worker pool 0 (chief)):

nbexecutor; --input-notebook=bucket/somedir_sample_notebook.ipynb; --output-notebook=bucket/somedir_sample_notebook.ipynb; --kernel-name=

**Comment by ijakas-syn** [comment]
I tried various configurations but the kernel-name value on Vertex is always empthy.

![image](https://github.com/google-github-actions/run-vertexai-notebook/assets/81167673/1b002359-a3c4-4909-81f3-50900e9e8947)
